tests/namespace_test/namespace_test2_generated.ts

```diff
@@ -1,6 +1,6 @@
 // automatically generated by the FlatBuffers compiler, do not modify
 
-import * as NS4989953370203581498 from "./namespace_test1_generated";
```

Comment:
Probably because of the differences in file paths and/or hashing algorithms in various std libraries.